packages/algob/src/internal/core/config/config-validation.ts

```diff
@@ -35,7 +35,7 @@ const ChainType = z.object({
   throwOnCallFailures: z.boolean().optional(),
   loggingEnabled: z.boolean().optional(),
   initialDate: z.string().optional()
-}).nonstrict();
+}).passthrough();
```

Comment:
what's the difference b/w `nonstrict` & `passthrough`?

Reply:
I think it same, but nonstrict is deprecated and zod recommend use passthrough